src/mumble/OpusCodec.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2005-2018 The Mumble Developers. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file at the root of the
+// Mumble source tree or at <https://www.mumble.info/LICENSE>.
+
+#ifndef MUMBLE_MUMBLE_OPUSCODEC_H_
+#define MUMBLE_MUMBLE_OPUSCODEC_H_
+
+#include <opus.h>
+
+#include <QtCore/QLibrary>
+
+#ifndef Q_OS_WIN
+#define __cdecl
+#endif
+
+// Loads Opus from a shared library and acts as a wrapper for its functions.
+class OpusCodec {
```

Comment:
Should be doxygen style comment `///`

https://wiki.mumble.info/wiki/Coding_Guidelines#Classes_.2F_Functions